src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsSlotsMercury.java

```diff
@@ -71,6 +73,46 @@ public void adsLeaderboardAndInContentOnPage(String wikiName,
   @Test(
       groups = "AdsSlotsMercury",
       dataProviderClass = MobileAdsDataProvider.class,
+      dataProvider = "leaderboardSlotOnPageWithInfobox"
+  )
+  public void adsLeaderboardOnPageWithInfobox(String wikiName,
+                                               String article,
+                                               String adUnit) {
+
+    String testedPage = urlBuilder.getUrlForPath(wikiName, article);
+    MobileAdsBaseObject ads = new MobileAdsBaseObject(driver, testedPage);
+
+    ads.verifyGptIframe(adUnit, MOBILE_TOP_LEADERBOARD, SRC);
+
+    Point leaderboardLocation = driver.findElement(By.id(MOBILE_TOP_LEADERBOARD)).getLocation();
+    Point infoboxLocation = driver.findElement(By.className("portable-infobox")).getLocation();
```

Comment:
Please don't use _low-level_ `findElement` in the test methods.
(one idea is to create `getElementLocationByCssSelector()` inside some `xxxObject` class and use it 4 times)